extensions/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/metrics/JvmMetricsTest.java

```diff
@@ -31,6 +32,7 @@
 /**
  * Validate all JVM metrics being produced.
  */
+@DisabledOnSemeru(reason = "JFR is not supported on Semeru yet")
```

Comment:
TBH, I don't understand what's going on because I had them passing. See `JdkUtil.isSemeru()` call below. Did we add some metrics recently?

Reply:
OK, let's try another approach first: https://github.com/quarkusio/quarkus/pull/51296 .

I will close this one but please don't drop the branch as we might reconsider this decision if ^ is not enough.